stories/module-pagespeed-insights-components.stories.js

```diff
@@ -215,28 +259,4 @@ storiesOf( 'PageSpeed Insights Module/Components', module )
 				<DashboardPageSpeedWidget { ...widgetComponentProps } />
 			</WithTestRegistry>
 		);
-	} )
-	.add( 'Dashboard widget (CTA)', () => {
```

Comment:
This story is no longer relevant as `DashboardPageSpeedWidget` displays nothing when `pagespeed-insights` is not active.